lib/record_store/record/alias.rb

```diff
@@ -2,7 +2,11 @@ module RecordStore
   class Record::ALIAS < Record
     attr_accessor :alias
 
-    validates :alias, presence: true, format: { with: Record::CNAME_REGEX, message: 'is not a fully qualified domain name' }
+    validates :alias, presence: true, format:
```

Comment:
Maybe, just maybe, I would put the opening of the bracket on the same line as `format`, as it would be clearer that it's a follow-up to that line. Would rubocop complain about that ?

Reply:
Yeah, it complains with `Layout/IndentFirstHashElement`.